csrc/ir/nodes.cpp

```diff
@@ -6059,6 +6059,189 @@ IterDomain* GroupedMmaOp::getGroupDimOfOutput() const {
 
 NVFUSER_DEFINE_CLONE_AND_CREATE(GroupedMmaOp)
 
+ScaledMmaOp::ScaledMmaOp(
```

Comment:
Should this be called ScaledGroupedMmaOp, or even better QuantizedGroupedMmaOp? I assume this op do only sub-16 matmuls.

Reply:
Maybe it's indeed doing matmul instead of grouped matmul. I might be confused by the following comment. 

> //! This operation performs a grouped matrix multiplication.

Reply:
This is NOT the grouped version. (That's the old PR with `GroupedMmaOp`, which also supports quantized version).

Going back to `Scaled` vs `Quantized`, I thought those two are used interchangeably. I'm neutral on that and am more than happy to rename it.
The API itself doesn't imply sub-16 bits dtype. But in reality we are only supporting that for now.